addOns/postman/postman.gradle.kts

```diff
@@ -32,6 +32,7 @@ crowdin {
 
 dependencies {
     zapAddOn("commonlib")
+    implementation("com.fasterxml.jackson.core:jackson-databind:2.12.5")
```

Comment:
Giving that this is going to be fixed shortly I don't think it's necessary to add it now.